node/src/blockchain/blockchain_interface.rs

```diff
@@ -122,14 +126,17 @@ impl Display for PayableTransactionError {
                 msg,
                 comma_joined_stringifiable(hashes, |hash| format!("{:?}", hash))
             )),
-            Self::UninitializedBlockchainInterface => Left("Uninitialized blockchain interface"),
+            Self::UninitializedBlockchainInterface => Left(
+                "The parameter blockchain-service-url \
+            must be set for scanners to function properly",
```

Comment:
The value of this message should be informed by the value chosen for the message above.